validation/foundry-config.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/contracts-bedrock/foundry.toml b/packages/contracts-bedrock/foundry.toml
+index 3d11af94b..c3d441735 100644
+--- a/packages/contracts-bedrock/foundry.toml
++++ b/packages/contracts-bedrock/foundry.toml
+@@ -8,7 +8,7 @@ remappings = [
+   '@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/',
+   '@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/',
+   '@rari-capital/solmate/=node_modules/@rari-capital/solmate',
+-  "@cwia/=node_modules/clones-with-immutable-args/src",
++  "@cwia/=node_modules/clones-with-immutable-args",
```

Comment:
This patch file is needed to compile contracts in the monorepo at the specified commit.